packages/ui-extensions/docs/surfaces/admin/staticPages/admin-extensions.doc.ts

```diff
@@ -71,13 +71,13 @@ const data: LandingTemplateSchema = {
       type: 'Generic',
       anchorLink: 'gettingStarted',
```

Comment:
could we make this lowercased? The sidebar lowercases the links and it breaks
```suggestion
      anchorLink: 'gettingstarted',
```